_posts/plotly_js/maps/density-mapbox/2019-08-16-earthquack_density.html

```diff
@@ -0,0 +1,28 @@
+---
+name: Outdoors Tile that Requires Token
+plot_url: https://codepen.io/plotly/embed/xxKEqyL/?height=514&theme-id=15263&default-tab=result
+language: plotly_js
+suite: density-maps
+order: 3
+sitemap: false
+arrangement: horizontal
+
+---
+Plotly.d3.csv('https://raw.githubusercontent.com/plotly/datasets/master/earthquakes-23k.csv',
+  function(err, rows){function unpack(rows, key) {return rows.map(function(row){ return row[key];
+})};
+
+var data = [{
+  lon: unpack(rows, 'Longitude'), lat: unpack(rows, 'Latitude'), radius:10,
+  z: unpack(rows, 'quakes.Magnitude'), type: "densitymapbox", coloraxis: 'coloraxis',
```

Comment:
Right now the colorbar is messed up because all the z data is undefined.
```suggestion
  z: unpack(rows, 'Magnitude'), type: "densitymapbox", coloraxis: 'coloraxis',
```